webapp/channels/src/components/advanced_create_comment/advanced_create_comment.tsx

```diff
@@ -838,6 +843,16 @@ class AdvancedCreateComment extends React.PureComponent<Props, State> {
         const ctrlAltCombo = Keyboard.cmdOrCtrlPressed(e, true) && e.altKey;
         const shiftAltCombo = !Keyboard.cmdOrCtrlPressed(e) && e.shiftKey && e.altKey;
 
+        // fix for FF not capturing the paste without formatting event when using ctrl|cmd + shift + v
```

Comment:
If this is a FF only fix, does it make sense to gate this with a browser check? Or is it safe to do this for all browsers?

Reply:
it doesn't hurt to do it for all browsers, I thought in doing the extra check, but it is an unnecessary extra validation.